### ISSUE TITLE
Enhancement: Overwrite ReadOnly Files

### ISSUE THREAD

**Issue by ferventcoder** [chucknorris/dropkick#25]
This is really annoying when trying to deploy.

Take a look at the sample code and determine if it can be beefed up to meet this need.

``` c#
           foreach (string file in Directory.GetFiles("\\"))
           {
               var ft = new FileInfo(file);
               ft.IsReadOnly = false;
           }
```

**Comment by ferventcoder** [comment]
One could also look at doing just removing the read only file attribute instead: http://www.csharp-examples.net/file-attributes/ and http://imak47.wordpress.com/2009/01/15/how-to-remove-readonly-attribute-from-a-file/ & http://www.csharptalk.com/2010/12/c-file-operations-delete-file-attribute.html and http://bytes.com/topic/c-sharp/answers/269366-how-remove-read-only-file-remote-server

**Comment by ferventcoder** [comment]
Fixed in d28b77a8acbca0b2a912582602a455632e1ca2de
